opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "ocsigen-start"
-version: "3.0.0"
```

Comment:
Pour un breaking change il suffit d'augmenter la version mineur : 3.1.0

Reply:
(Si je ne me trompe pas. C'est peut-être différent pour Ocsigen.)

Reply:
What warrants a major version then?

Reply:
Some sort of paradigm shift for example? Again, I vaguely remember @balat saying that for Ocsigen every breaking change requires an increase of the major version.

Reply:
https://semver.org/
> Given a version number MAJOR.MINOR.PATCH, increment the:
>
>    MAJOR version when you make incompatible API changes,
>    MINOR version when you add functionality in a backwards compatible manner, and
>    PATCH version when you make backwards compatible bug fixes.